### PR TITLE
[Doc bash][Add Image to trust authors in VSC and remove state folders]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e5b29499f33e8df149a093216096cc1b2021ee76/msteams-platform%2Fsbs-gs-bot.yml) | 91 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd6ca634-94f9-4254-b80a-315a51ae2f65) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/dda9b2b8862b24155cf4d7d1cdc9a30bc67cfa2d/msteams-platform%2Fsbs-gs-bot.yml) | 91 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a17b3a7b-fd88-4ad8-b9fa-080d1ef48e0c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/0adc553a1dc525357cc4eba8c28da661f1d828f0/msteams-platform%2Fsbs-gs-bot.yml) | 91 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cd03b6bc-51c7-460c-ab3a-4a1badbf4a35) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-bot.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e54b7a7232f2d2385a9650160251d071be0635df/msteams-platform%2Fsbs-gs-bot.yml) | 91 | 37 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7ac67e30-d129-4706-b85f-bf3d83a4fc8c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-bot.yml [hunk 1]
```diff
@@ -206,6 +206,10 @@ items:
 
         :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/first-bot/enter-name-bot.png" alt-text="Screenshot showing where to enter the app name.":::
 
+        A dialog appears, where you would be required to choose yes or no to trust the authors of the files in this folder.
```
Comment (Wajeed-msft) [inline msteams-platform/sbs-gs-bot.yml:209]: Should we also mention that user need to select Yes to move forward?

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**